artemis-commons/src/main/java/org/apache/activemq/artemis/utils/DefaultSensitiveStringCodec.java

```diff
@@ -20,6 +20,8 @@
 import javax.crypto.SecretKeyFactory;
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.SecretKeySpec;
+import javax.naming.ConfigurationException;
```

Comment:
I don't think this is the proper exception to use here as this method has nothing to do with naming (e.g. JNDI).